docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -318,10 +317,34 @@ eigenlayer operator status operator.yaml
 
 ---
 
-### Metadata Updates
+## Metadata Updates
 
 This is the command you can use to make changes or updates to the metadata of your operator.
 
 ```
 eigenlayer operator update operator.yaml
 ```
+
+
+
+## delegationApprover Design Patterns
+
+Delegation Approver functionality can be used in multiple ways to give Operators additional programmatic control over which Restakers they accept delegation from.
+
+
+### EOA Signature Approval
+
+One series of designs involves passing a unique signature from the Operator to the Restaker requesting approval. The unique signature will have a corresponding ‘salt’ (unique value used once) and an ‘expiry’. The Restaker passes the signature (salt & expiry) into the `DelegationManager.delegateTo` function ([source here](https://github.com/Layr-Labs/eigenlayer-contracts/blob/mainnet/src/contracts/core/DelegationManager.sol#L135-L155)). This function uses EIP1271 to check the signature, so either:
+- A) The Operator has set an EOA as their `delegationApprover` and the DelegationManager simply checks that the signature is a valid ECDSA signature from the EOA.
+- OR B) The Operator has set a smart contract as their `delegationApprover` and the DelegationManager calls the isValidSignature function on the `delegationApprover` and checks if the contract returns `0x1626ba7e` (as defined in the [EIP-1271 specification](https://eips.ethereum.org/EIPS/eip-1271#specification)).
+
+
+**Whitelist and Blacklisting Restakers for Delegation**
```

Comment:
for parallel structure, recommend 'whitelisting' instead of 'whitelist'

Reply:
good call, done